clients/java/src/main/java/io/camunda/client/api/search/response/ProcessInstanceState.java

```diff
@@ -24,8 +24,8 @@ public enum ProcessInstanceState {
   TERMINATED,
   UNKNOWN_ENUM_VALUE;
 
-  public static ProcessInstanceStateEnum toProtocolState(final ProcessInstanceState value) {
-    return (value == null) ? null : ProcessInstanceStateEnum.fromValue(value.name());
+  public ProcessInstanceStateEnum toProtocolState() {
```

Comment:
❓ Can we leave those changes and the ones related to this out of this PR? That will be tackled by yhttps://github.com/camunda/camunda/pull/29792 already and will avoid unnecessary merge conflicts there when this is merged here.

Reply:
Sure, I'll drop them.